resources/templates/netCore/Dockerfile.template

```diff
@@ -12,11 +12,15 @@ ENV ASPNETCORE_URLS=http://+:{{ ports.[0] }}
 {{/if}}
 {{#unless (isRootPort ports)}}
 {{#if (eq netCorePlatformOS 'Linux')}}
+{{#if netCoreBaseImageDefaultUser }}
+USER {{ netCoreBaseImageDefaultUser }}
+{{else}}
 # Creates a non-root user with an explicit UID and adds permission to access the /app folder
 # For more info, please refer to https://aka.ms/vscode-docker-dotnet-configure-containers
 RUN adduser -u 5678 --disabled-password --gecos "" appuser && chown -R appuser /app

```

Comment:
Does `adduser` still work on newer Debian but .NET 7 (and below)?

Reply:
No, `adduser` will not work on Debian with .NET 7 or below. This is the error I get from running the `adduser` command from .NET 6 scaffolding with a Debian 12 image: `adduser: not found`

But in this case, since all .NET 7 images are based on Debian 11 and below, the case you described theoretically wouldn't happen.

Reply:
That's what I'm concerned about--will .NET 7 images get updated to Debian 12 at some point?

Reply:
I agree, it's definitely concerning. We'll have to reach out to the Docker image team to get a sure answer.